src/Controller/Profiles/IndexController.php

```diff
@@ -15,6 +15,7 @@ class IndexController extends BaseController
     public function __invoke(CoreEntity $coreEntity, ProfileService $isiteService)
     {
         $this->setIstatsProgsPageType('profiles_list');
+        $this->setAtiContentType('list-datadriven');
```

Comment:
```suggestion
        $this->setAtiContentType('list-profiles');
```
Was this supopse to be like that?

Reply:
Yup, changed.